lib/terrafying/components/loadbalancer.rb

```diff
@@ -216,9 +216,14 @@ def autoscale(set, target_value:, disable_scale_in:)
       end
 
       def make_identifier(type, vpc_name, name)
-        gen_id = "#{type}-#{tf_safe(vpc_name)}-#{name}"
-        return Digest::SHA256.bubblebabble(gen_id)[0..15] if @hex_ident || gen_id.size > 26
 
+        gen_id = "#{type}-#{vpc_name}-#{name}"
+        hex = Digest::SHA2.hexdigest(gen_id)[0..24]
+        if hex[0..0] =~ /[a-z]/
```

Comment:
is it possible for there to ever be a capital letter? could match with `[a-zA-Z]` in the regex if so

Reply:
from the look of hexdigest, it'll only output lowercase.